site/content/en/docs/Guides/metrics.md

```diff
@@ -174,6 +176,48 @@ Open a web browser to [http://127.0.0.1:3000](http://127.0.0.1:3000), you should
 
 > Makefile targets `make grafana-portforward`,`make kind-grafana-portforward` and `make minikube-grafana-portforward`.
 
+### Stackdriver installation
+
+In order to use [Stackdriver monitoring](https://app.google.stackdriver.com) you should enable Stackdriver Monitoring API on Google Cloud Console. You need to grant all the necessary permissions to the users (see [Access Control Guide](https://cloud.google.com/monitoring/access-control)). Stackdriver exporter uses a strategy called Application Default Credentials (ADC) to find your application's credentials. Details could be found here [Setting Up Authentication for Server to Server Production Applications](https://cloud.google.com/docs/authentication/production).
+
+Note that Stackdriver monitoring is enabled by default on GKE clusters, however you can follow this [guide](https://cloud.google.com/kubernetes-engine/docs/how-to/monitoring#enabling_stackdriver_monitoring) if it was disabled on your GKE cluster.
+
+Default metrics exporter is Prometheus. In order to change it to Stackdriver update following helm parameters in {{< ghlink href="/install/helm/agones/values.yaml" branch="master" >}}values file{{< /ghlink >}}:
```

Comment:
@Kuqd is this the right way to tell people to do this?  Or should we be directing people to the command line arguments?

Reply:
Helm users should know how set their values, I would give hint for that and not for modifying our own default values files.

Reply:
@markmandel I need to update this, but lets wait Cyril